part-1-ajax/test.js

```diff
@@ -1,76 +1,107 @@
+
+
+
 function getCountryName(code) {
+  var countryCode = code[0].alpha3Code
```

Comment:
Thanks Charles.  As long as the code works as intended right? Optimization
can come after.

On Sep 9, 2016 2:45 PM, "Charles Kuang" <notifications@github.com> wrote:

> In part-1-ajax/test.js
> <https://github.com/C4Q/ac3.1-exam-4/pull/24#discussion_r78229741>:
>
> >  function getCountryName(code) {
> > +  var countryCode = code[0].alpha3Code
>
> good clean code. nicely done jung
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/C4Q/ac3.1-exam-4/pull/24/files/7324672043529e8b01f70cff04b857d0c167e431#r78229741>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATauGCyV4lrpmlP2YJDsUHgCF4qWyWhxks5qoalPgaJpZM4J4crl>
> .
>